src/ipyradiant/visualization/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""vis widgets
+"""
+__all__ = ["CytoscapeVisualization"]
+from .vis_tool import CytoscapeVisualization
```

Comment:
maybe we rename this file as well, e.g. `cytoscape`. wanna be careful not to name things that are already packages, but i think since it's `ipyctoscape` we'll be fine.

Reply:
you're talking about the vis_tool file right?

Reply:
yeah. what happens when you add the next tool?